### ISSUE TITLE
Version en about prosus wallet

### ISSUE THREAD

**Issue by rigoy22k** [ProsusCorp/prosus-gui#1]
Sería ideal tener el numero de version en uso de la wallet 
![image](https://user-images.githubusercontent.com/11945557/37176401-0b121430-22fb-11e8-8763-16b918edc313.png)

**Comment by yerkobits** [comment]
Sí. Lo omiti porque la prosus-gui iba desfasada con respecto al prosus-daemon.
A partir de 0.5.3 se compilan unificadamente.
